httpx/_decoders.py

```diff
@@ -99,37 +95,44 @@ class BrotliDecoder(ContentDecoder):
     """
 
     def __init__(self) -> None:
-        if brotlicffi is None:  # pragma: nocover
+        if brotli is None:  # pragma: nocover
             raise ImportError(
-                "Using 'BrotliDecoder', but the 'brotlicffi' library "
-                "is not installed."
+                "Using 'BrotliDecoder', but neither of the 'brotlicffi' or 'brotli' "
+                "packages have been installed. "
                 "Make sure to install httpx using `pip install httpx[brotli]`."
             ) from None
 
-        self.decompressor = brotlicffi.Decompressor()
+        self.decompressor = brotli.Decompressor()
         self.seen_data = False
         if hasattr(self.decompressor, "decompress"):
-            self._decompress = self.decompressor.decompress
+            # The 'brotlicffi' package.
+            self._decompress = self.decompressor.decompress  # pragma: nocover
         else:
+            # The 'brotli' package.
             self._decompress = self.decompressor.process  # pragma: nocover
```

Comment:
I think`brotlicffi`  also has a `process` alias for `decompress`:

https://github.com/python-hyper/brotlicffi/blob/68c0431673e6c59549079e6588b043a707262c80/src/brotlicffi/_api.py#L423

How about just use `self.decompressor.process`?

Reply:
True. Tho I double checked and the `brotlipy` package doesn't, so what we want to do here depends on if we want to take care around the possibility that on some systems the older `brotlipy` package might have been installed. (Or not.)

If we do care about that awkward case, then we should just update the comment strings here.
If we don't case about that case, then we should just always use `Decompressor().process()`.

Perhaps we could check which of `brotli` vs `brotlipy` is installed the `_compat.py` module, and raise an error if `brotlipy` is installed and is shadowing the `brotli` namespace.

For example:

```python
    if hasattr(brotli.Decompressor, 'decompress'):
        raise RuntimeError(
            "The brotlipy library appears to be installed, and is shadowing the 'brotli' namespace. "
            "Uninstall brotlipy and use brotlicffi instead."
        )
```